src/test/regress/expected/multi_explain.out

```diff
@@ -1106,21 +1106,11 @@ EXPLAIN (COSTS FALSE, FORMAT YAML)
             Dependent Jobs:
               - Map Task Count: 2
                 Merge Task Count: 1
--- test parallel aggregates
-SET parallel_setup_cost=0;
-SET parallel_tuple_cost=0;
-SET min_parallel_relation_size=0;
-ERROR:  unrecognized configuration parameter "min_parallel_relation_size"
```

Comment:
The intention here seems to be to confirm that local explain plans still work when Citus hooks are present, so parallelism seems unnecessary. Also, errors.